abmarl/sim/gridworld/agent.py

```diff
@@ -40,47 +63,64 @@ def position(self, value):
         self._position = value
 
     @property
-    def encoding(self):
+    def view_blocking(self):
         """
-        The numerical value given to other agents who observe this agent.
+        Specify if this agent blocks other agent's observations.
         """
-        return self._encoding
+        return self._view_blocking
 
-    @encoding.setter
-    def encoding(self, value):
-        assert type(value) is int, f"{self.id}'s encoding must be an integer."
-        assert value != -2, "-2 encoding reserved for masked observation."
-        assert value != -1, "-1 encoding reserved for out of bounds."
-        assert value != 0, "0 encoding reserved for empty cell."
-        self._encoding = value
+    @view_blocking.setter
+    def view_blocking(self, value):
+        assert type(value) is bool, "View blocking must be either True or False."
+        self._view_blocking = value
+
+    @property
+    def overlappable(self):
+        """
+        Agents may be able to overlap each other.
+
+        Overlappable agents can overlap other overlappable agents if the Simulation
```

Comment:
Remove the part about "if using an overlappable grid" since there is only one grid now.